spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -59,7 +59,7 @@ trait CommonJobExecution {
   protected val menasBaseUrls: List[String] = MenasConnectionStringParser.parse(conf.getString("menas.rest.uri"))
 
   protected def obtainSparkSession[T](jobName: String)(implicit cmd: JobConfigParser[T]): SparkSession = {
-    val enceladusVersion = ProjectMetadata.enceladusVersion
+    val enceladusVersion = ProjectMetadata.enceladusVersion()
```

Comment:
Doesn't this stop providing correct number if utils ever become a different version? Why not just extend the trait with `ProjectMetadata`?

Reply:
The idea is to extend only in Menas. I would say(although not 100% sure) that it will not provide a different result cause the result of that function is dependent on the menas pom configuration, it mostly just calls that code